swapvenue/binance/binance_swap_venue.go

```diff
@@ -123,7 +123,8 @@ func (b *BinanceSwapVenue) GetPrice(ctx context.Context, pair swapvenuetypes.Swa
 	url := fmt.Sprintf("%s/ticker/price?symbol=%s", b.config.URL, baseQuote)
 
 	var binancePriceResponse binancePriceResponse
-	if err := httputil.RunGet(ctx, url, nil, &binancePriceResponse); err != nil {
+	_, err := httputil.MakeRequest(ctx, httputil.HttpGET, url, nil, nil, &binancePriceResponse)
```

Comment:
Let's create a wrapper API such as `httputil.Get(...)` so that clietns don't have to provide `htttputil.HttpGET`. Same for POST

Reply:
https://github.com/osmosis-labs/osmoutil-go/pull/3/commits/adcd11fdc0ef1f0273ab40fbbc99783ea12d8ec9